### PR TITLE
Bugfixes.

### PR DESCRIPTION
**Pull request by sunipkm**
Fixed a bug where executing pyiri90.runiri() would modify CWD. Fixed another issue where the data files were not found.

### CODE REVIEW

File: pyiri90/runiri90.py [hunk 1]
```diff
@@ -18,12 +18,18 @@ def runiri(dt,z,glat,glon,f107,f107a,ap,mass=48):
...
+        chdir(pyiri90.__path__[0])
+        logging.debug(getcwd())
+        outf,oarr = iri90.iri90(JF,jmag,glat,glon % 360., -f107,
+                        dt.strftime('%m%d'),
+                        dt.hour+dt.minute//60+dt.second//3600,
+                        z,'data/')
+    except Exception as e:
+        chdir(cwd)
+        raise e
+    chdir(cwd)
```
Comment (georgegeddes) [inline pyiri90/runiri90.py]: This can be simplified to:
```python
    finally:
        chdir(cwd)
```

**Review by georgegeddes — CHANGES_REQUESTED** [review]
Thanks for the fix! I had to change the setup script to get this to install properly. Specifically, line 8 of setup.py should be `iridata = glob(join('pyiri90', 'data','*.asc'))` to find the data files.

This fork is far behind the upstream and I haven't used that in a long time so I'm not sure if this bug exists there too. If it does, you should submit a pr there too.

File: pyiri90/runiri90.py (hunk 1 above)
Comment (sunipkm) [reply pyiri90/runiri90.py]: Right, `finally` is so much easier. I use Python exceptions coming from a very C++ paradigm lol

I believe upstream iri90 has been using executable binaries lately and using subprocess + stdout capture to do things. Not sure I like that approach. I will make those changes. Thanks man.

Comment (sunipkm) [reply pyiri90/runiri90.py]: and I just applied the fixes.

File: pyiri90/runiri90.py [hunk 2]
```diff
@@ -4,51 +4,72 @@
...
+        f107 (float): F10.7 index for the day.
+        f107a (float, optional): 81-day average F10.7 index, unused. Present to maintain parity with GLOW calls. Defaults to None.
+        ap (float, optional): Global ap index, unused. Present to maintain parity with GLOW calls. Defaults to None.
+        jmag (bool, optional): Geographic (False) or magnetic (True) latitude, longitude. Defaults to False.
+        JF (ndarray, optional): Flags for the model. Defaults to array((1, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 0), bool) for IRI version found in Solomon (1993). Use (1,1,1) + (0,0,0) +(1,)*14 + (0,1,0,1,1,1,1,0,0,0,1,1,0,1,0,1,1,1) for 2013 version of IRI.
+
+    Returns:
+        Tuple[DataFrame, ndarray]: DataFrame containing density and temperature profiles, array containing additional info -> [nmF2 (m-3), hmF2 (km), nmF1 (m-3), hmF1 (km), nmE (m-3), hmE (km), nmD (m-3), hmD (km), hhalf (km), B0 (km), vally base, valley top, Te-peak (K), Te peak height (km), Te-Mod (300 km), Te-Mod (400 km), Te-Mod (600 km), Te-Mod (1400 km), Te-Mod (3000 km), Te-Mod (120 km) = Tn = Ti, Te-Mod (430 km), Z (where Te=Ti), SZA (deg), Sun decl (deg), dip, dip lat, modified dip lat]
+    """
```
Comment (georgegeddes) [inline pyiri90/runiri90.py:36]: nice documentation!

**Review by georgegeddes — APPROVED** [review]

File: pyiri90/runiri90.py (hunk 2 above)
Comment (sunipkm) [reply pyiri90/runiri90.py:36]: Thank you sir!

**Pull request merged**